src/RestApi/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -242,6 +248,31 @@ public function get_item_response( $cart_item ) {
 		];
 	}
 
+	/**
+	 * Generate a summary from the product short/full description.
+	 *
+	 * @param \WC_Product $product Product instance.
+	 * @return string
+	 */
+	protected function get_product_summary( \WC_Product $product ) {
+		$short_description = $product->get_short_description();
+
+		if ( $short_description ) {
+			return \wc_format_content( $short_description );
```

Comment:
~We filtered this originally (`woocommerce_short_description`), are we not going to filter this anymore?  Any concern with folks in the wild that might have already implemented this filter (unlikely but still possible)?~

Edited: I see that `wc_format_content` applies that filter 👍